### ISSUE TITLE
Can not create new app and generate component in 6.0.4

### ISSUE THREAD

**Issue by NuttapongPun** [angular/angular#24120]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior

I'm update angular and run ..
<pre><code>ng new my-app</code></pre>
But it not generate my-app and show this massage in terminal
<pre><code>
Schematic input does not validate against the Schema: {"dryRun":false,"version":"6.0.4","skipGit":false,"skipInstall":false,"linkCli":false,"commit":true,"newProjectRoot":"projects","inlineStyle":false,"inlineTemplate":false,"routing":false,"prefix":"app","style":"css","skipTests":false}
Errors:

  Data path "" should NOT have additional properties(dryRun).
</code></pre>

Note1: I try to create component and module in exist app but it show same ERROR.
Note2: It happen since i update angular/cli, I try remove and install version 6.0.3,
 It working but auto update when i create new app

## Environment
<pre><code>
     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 6.0.4
Node: 10.2.1
OS: darwin x64
Angular: 
... 

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.6.4
@angular-devkit/core         0.6.4
@angular-devkit/schematics   0.6.4
@schematics/angular          0.6.4
@schematics/update           0.6.4
rxjs                         6.2.0
typescript                   2.7.2
</code></pre>

**Comment by heavy200t** [comment]
Got the same issue.

**Comment by IgorMinar** [comment]
I was able to repro this issue

```
$ yarn global add @angular/cli@latest
yarn global v1.3.2
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
success Installed "@angular/cli@6.0.4" with binaries:
      - ng
✨  Done in 5.93s.
✔ ~/Development/tmp [master ↓·1↑·119|…2⚑ 3] 
22:13 $ ng new cli-6.0.4-test
Schematic input does not validate against the Schema: {"dryRun":false,"version":"6.0.4","skipGit":false,"skipInstall":false,"linkCli":false,"commit":true,"newProjectRoot":"projects","inlineStyle":false,"inlineTemplate":false,"routing":false,"prefix":"app","style":"css","skipTests":false}
Errors:

  Data path "" should NOT have additional properties(dryRun).
```

**Comment by DinhKhoi** [comment]
same issue

**Comment by DinhKhoi** [comment]
i got same issue.

**Comment by DinhKhoi** [comment]
![image](https://user-images.githubusercontent.com/33003395/40527273-af6dc2d4-6015-11e8-8152-37896011780d.png)

![image](https://user-images.githubusercontent.com/33003395/40527286-bf76be60-6015-11e8-9ad4-7aacbc11554c.png)

npv -v
tsc -v
node -v 
 working perfectly except "ng new my-app"

**Comment by DinhKhoi** [comment]
I found solution:  just run ' npm install -g @angular/cli ' again and it work like magic

**Comment by hansl** [comment]
We just released 6.0.5 that is a rollback to 6.0.3.

@brocco the culprit is https://github.com/angular/devkit/pull/899 (along with the CLI adding properties to the schematics input it shouldn't). Let's find a fix for next week.

I'm blocking the release for this week.

**Comment by hansl** [comment]
Closing this issue as fixed with the rollback.
